app/models/task.py

```diff
@@ -2,4 +2,35 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String(80))
+    description = db.Column(db.String(255))
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
```

Comment:
Fun fact: the default value for nullable is `True `so we don't need to explicitly add it!